### PR TITLE
LegacyMakerOracleProxy to migrate old Proxy

### PR DESCRIPTION
**Pull request by felix2feng**

### CODE REVIEW

File: contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+
+/**
+ * @title LegacyMakerOracleProxy
+ * @author Set Protocol
+ *
+ * Permissioned oracle proxy that is intended to replicate the interface of the MakerDAO Legacy IMedian
+ * oracle that returns bytes32 values.
+ */
+contract LegacyMakerOracleProxy is
```
Comment (bweick) [inline contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol:31]: At this point it's not touching Maker's oracles so this name feels like a bit of a misnomer. This contract is more to convert our OracleProxies to interact with our legacy contracts.

File: contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    )
+        public
+    {
+        oracleInstance = _oracleAddress;
+    }
+
+    /* ============ External ============ */
+
+    /*
+     * Reads value of external oracle and passed to Set system. Only authorized addresses are allowed
```
Comment (bweick) [inline contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol:61]: Not really gonna be reading anything external to our system, more converting within our system to Legacy contracts

File: contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol (hunk 1 above)
Comment (bweick) [reply contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol:31]: Maybe `LegacyContractOracleAdapter` or something like that is better

Comment (felix2feng) [reply contracts/meta-oracles/proxies/LegacyMakerOracleProxy.sol:31]: Maybe `MedianOracleAdapter` since we're trying to conform to `IMedian`

**Pull request merged**